src/styles/blocks/card-product.scss

```diff
@@ -0,0 +1,81 @@
+.card-product {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: left;
+  width: $cardWidth;
+  padding: 32px 16px 16px;
+  border-radius: $borderRadius;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    padding-top: 40px;
+    margin: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $mainTextColor;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $additionalTextColor;
+    margin: 0;
+    padding: 4px 0 16px;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price {
+    font-size: 12px;
+    margin: 0;
+    line-height: 18px;
+    color: $additionalTextColor;
+  }
+
+  &__cost {
+    margin: 0;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $mainTextColor;
+  }
+
+  &__button {
+    width: 100%;
+    height: 40px;
+    margin-top: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+    background-color: $btnColor;
+    border-radius: $borderRadius;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: #fff;
```

Comment:
Consider using variables for repeating colors

Reply:
Variables added